test/sv/interfaces.mlir

```diff
@@ -35,6 +35,19 @@ func @test() {
   // CHECK-NEXT:   sv.interface.modport @dataflow_out ("output" @data, "output" @valid, "input" @ready)
   // CHECK-NEXT: }
 
-  // CHECK-NEXT: return
-  return
+  rtl.externmodule @Rcvr (%m: !sv.modport<@handshake_example::@dataflow_in>)
+  // CHECK-LABEL: rtl.externmodule @Rcvr(!sv.modport<@handshake_example::@dataflow_in> {rtl.name = "m"})
+
+  rtl.module @Top () {
+    %iface = sv.interface.instance : !sv.interface<@handshake_example>
+    %ifaceInPort = sv.modport.get %iface @dataflow_in :
+      !sv.interface<@handshake_example> -> !sv.modport<@handshake_example::@dataflow_in>
+    // This next line may or may not make sense depending on how we decide to
+    // model interactions with the RTL dialect.
```

Comment:
IMHO this makes sense, and I think there is a path forward to supporting this in EmitVerilog.cpp that is also relatively sane.